2015/08/12 14:02:10 [error] [exception.CException] exception 'CException' with message 'Property "Questions.answer" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('answer')
#1 /var/www/yii/blog/protected/modules/questions/views/default/view.php(37): CActiveRecord->__get('answer')
#2 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/yii/bl...')
#3 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/yii/bl...', Array, true)
#4 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/yii/bl...', Array, true)
#5 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('view', Array, true)
#6 /var/www/yii/blog/protected/modules/questions/controllers/DefaultController.php(41): CController->render('view', Array)
#7 [internal function]: DefaultController->actionView('5')
#8 /var/www/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(DefaultController), Array)
#9 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(DefaultController), Object(ReflectionMethod), Array)
#10 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /var/www/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#13 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#17 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#19 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#20 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#21 /var/www/yii/blog/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/5
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/category/3
---
2015/08/12 14:02:11 [error] [exception.CException] exception 'CException' with message 'Property "Questions.answer" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('answer')
#1 /var/www/yii/blog/protected/modules/questions/views/default/view.php(37): CActiveRecord->__get('answer')
#2 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/yii/bl...')
#3 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/yii/bl...', Array, true)
#4 /var/www/yii/framework/web/CController.php(869): CBaseController->renderFile('/var/www/yii/bl...', Array, true)
#5 /var/www/yii/framework/web/CController.php(782): CController->renderPartial('view', Array, true)
#6 /var/www/yii/blog/protected/modules/questions/controllers/DefaultController.php(41): CController->render('view', Array)
#7 [internal function]: DefaultController->actionView('5')
#8 /var/www/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(DefaultController), Array)
#9 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(DefaultController), Object(ReflectionMethod), Array)
#10 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#11 /var/www/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#12 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#13 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#14 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#15 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#16 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#17 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#18 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#19 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#20 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#21 /var/www/yii/blog/index.php(13): CApplication->run()
#22 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/5
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:03:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Error Processing Request' in /var/www/yii/blog/protected/modules/questions/controllers/DefaultController.php:162
Stack trace:
#0 /var/www/yii/framework/web/actions/CInlineAction.php(49): DefaultController->actionAnswer()
#1 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#2 /var/www/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#3 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#4 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#5 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#6 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#7 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('answer')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#11 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#12 /var/www/yii/blog/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/blog/index.php/questions/answer
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/answer
---
2015/08/12 14:05:23 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Error Processing Request' in /var/www/yii/blog/protected/modules/questions/controllers/DefaultController.php:164
Stack trace:
#0 /var/www/yii/framework/web/actions/CInlineAction.php(49): DefaultController->actionAnswer()
#1 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#2 /var/www/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#3 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#4 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#5 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#6 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#7 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#8 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#9 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('answer')
#10 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#11 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#12 /var/www/yii/blog/index.php(13): CApplication->run()
#13 {main}
REQUEST_URI=/yii/blog/index.php/questions/answer
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/answer
---
2015/08/12 14:32:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/adverts/view/id/269".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/adver...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/adverts/view/id/269
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:32:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/adverts/view/id/269".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/adver...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/adverts/view/id/269
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:32:54 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/adverts/view/id/264".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/adver...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/adverts/view/id/264
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:33:00 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/adverts/view/id/264".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/adver...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/adverts/view/id/264
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:43:27 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/adverts/view/id/5".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/adver...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/adverts/view/id/5
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:50:31 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/id/10".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/id/10
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:50:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/view/id/8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/view/...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/id/8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/questions/view/id/10
---
2015/08/12 14:50:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/view/id/6".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/view/...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/id/6
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/id/8
---
2015/08/12 14:50:36 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/view/id/8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/view/...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/id/8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/id/6
---
2015/08/12 14:50:38 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/view/id/11".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/view/...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/id/11
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/id/8
---
2015/08/12 14:50:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/view/id/8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/view/...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/id/8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/id/6
---
2015/08/12 14:50:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/view/id/6".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/view/...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/id/6
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/id/8
---
2015/08/12 14:50:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/view/id/8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/view/...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/id/8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/questions/view/id/10
---
2015/08/12 14:50:43 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/id/10".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/id/10
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:50:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/id/10".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/id/10
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:51:11 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/5
---
2015/08/12 14:51:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view9".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view9')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view9')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#3 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/yii/blog/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/blog/index.php/questions/view9
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/questions/view8
---
2015/08/12 14:51:14 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view7".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view7')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view7')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#3 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/yii/blog/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/blog/index.php/questions/view7
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view9
---
2015/08/12 14:51:15 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view11".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view11')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view11')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#3 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/yii/blog/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/blog/index.php/questions/view11
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view7
---
2015/08/12 14:51:21 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "view11".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('view11')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view11')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#3 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/yii/blog/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/blog/index.php/questions/view11
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view7
---
2015/08/12 14:51:25 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/9".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/9
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/7
---
2015/08/12 14:51:26 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/9".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/9
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/11
---
2015/08/12 14:51:28 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/10".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/10
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/8
---
2015/08/12 14:51:30 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/7".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/7
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/9
---
2015/08/12 14:51:32 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/11".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/11
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/9
---
2015/08/12 14:51:39 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/10
---
2015/08/12 14:51:41 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/6".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/6
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/7
---
2015/08/12 14:51:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/6".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/6
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/6
---
2015/08/12 14:51:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/6".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/6
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/6
---
2015/08/12 14:52:13 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/11".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/11
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/6
---
2015/08/12 14:52:16 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/7".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/7
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/8
---
2015/08/12 14:54:10 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/7".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/7
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/8
---
2015/08/12 14:54:45 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/7".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/7
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/8
---
2015/08/12 14:54:50 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/9".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/9
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/11
---
2015/08/12 14:55:03 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/9".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/9
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/11
---
2015/08/12 14:55:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/9
---
2015/08/12 14:55:44 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/7
---
2015/08/12 14:55:47 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/8".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/8
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/9
---
2015/08/12 14:56:40 [error] [exception.CException] exception 'CException' with message 'Property "LastQuestions.params" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/blog/protected/components/LastQuestions.php(15): CComponent->__get('params')
#1 /var/www/yii/framework/zii/widgets/CPortlet.php(97): LastQuestions->renderContent()
#2 /var/www/yii/framework/web/CBaseController.php(206): CPortlet->run()
#3 /var/www/yii/blog/protected/views/layouts/main.php(73): CBaseController->endWidget()
#4 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/yii/bl...')
#5 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/yii/bl...', Array, true)
#6 /var/www/yii/framework/web/widgets/CContentDecorator.php(76): CBaseController->renderFile('/var/www/yii/bl...', Array, true)
#7 /var/www/yii/framework/web/widgets/CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#8 /var/www/yii/framework/web/widgets/COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#9 /var/www/yii/framework/web/CBaseController.php(206): COutputProcessor->run()
#10 /var/www/yii/framework/web/CBaseController.php(300): CBaseController->endWidget('CContentDecorat...')
#11 /var/www/yii/blog/protected/views/layouts/column1.php(6): CBaseController->endContent()
#12 /var/www/yii/framework/web/CBaseController.php(126): require('/var/www/yii/bl...')
#13 /var/www/yii/framework/web/CBaseController.php(95): CBaseController->renderInternal('/var/www/yii/bl...', Array, true)
#14 /var/www/yii/framework/web/CController.php(784): CBaseController->renderFile('/var/www/yii/bl...', Array, true)
#15 /var/www/yii/blog/protected/modules/questions/controllers/DefaultController.php(47): CController->render('view', Array)
#16 [internal function]: DefaultController->actionView('7')
#17 /var/www/yii/framework/web/actions/CAction.php(108): ReflectionMethod->invokeArgs(Object(DefaultController), Array)
#18 /var/www/yii/framework/web/actions/CInlineAction.php(47): CAction->runWithParamsInternal(Object(DefaultController), Object(ReflectionMethod), Array)
#19 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#20 /var/www/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#21 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#22 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#23 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#24 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#25 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#26 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#27 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('view')
#28 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#29 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#30 /var/www/yii/blog/index.php(13): CApplication->run()
#31 {main}
REQUEST_URI=/yii/blog/index.php/questions/view/7
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/questions/view/8
---
2015/08/12 14:57:42 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/9".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/9
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/7
---
2015/08/12 14:59:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/9".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/9
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/7
---
2015/08/12 14:59:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "questions/questions/view/9".' in /var/www/yii/framework/web/CWebApplication.php:286
Stack trace:
#0 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/quest...')
#1 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#2 /var/www/yii/blog/index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/yii/blog/index.php/questions/questions/view/9
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/7
---
2015/08/12 15:46:08 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "rightanswer".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('rightanswer')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('rightanswer')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#3 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/yii/blog/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/blog/index.php/questions/rightanswer
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/11
---
2015/08/12 15:46:17 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "rightanswer".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('rightanswer')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('rightanswer')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#3 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/yii/blog/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/blog/index.php/questions/rightanswer
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/11
---
2015/08/12 15:46:35 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'The system is unable to find the requested action "rightanswer".' in /var/www/yii/framework/web/CController.php:483
Stack trace:
#0 /var/www/yii/framework/web/CController.php(270): CController->missingAction('rightanswer')
#1 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('rightanswer')
#2 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#3 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#4 /var/www/yii/blog/index.php(13): CApplication->run()
#5 {main}
REQUEST_URI=/yii/blog/index.php/questions/rightanswer
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/11
---
2015/08/12 15:47:03 [error] [exception.CException] exception 'CException' with message 'Property "Answers.rigth_answer" is not defined.' in /var/www/yii/framework/base/CComponent.php:173
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(161): CComponent->__set('rigth_answer', '1')
#1 /var/www/yii/blog/protected/modules/questions/controllers/DefaultController.php(161): CActiveRecord->__set('rigth_answer', '1')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): DefaultController->actionRightAnswer()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#10 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('rightanswer')
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#13 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#14 /var/www/yii/blog/index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/blog/index.php/questions/rightanswer
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/11
---
2015/08/12 15:47:21 [error] [exception.CException] exception 'CException' with message 'Property "Answers.title" is not defined.' in /var/www/yii/framework/base/CComponent.php:130
Stack trace:
#0 /var/www/yii/framework/db/ar/CActiveRecord.php(145): CComponent->__get('title')
#1 /var/www/yii/blog/protected/modules/questions/controllers/DefaultController.php(163): CActiveRecord->__get('title')
#2 /var/www/yii/framework/web/actions/CInlineAction.php(49): DefaultController->actionRightAnswer()
#3 /var/www/yii/framework/web/CController.php(308): CInlineAction->runWithParams(Array)
#4 /var/www/yii/framework/web/filters/CFilterChain.php(133): CController->runAction(Object(CInlineAction))
#5 /var/www/yii/framework/web/filters/CFilter.php(40): CFilterChain->run()
#6 /var/www/yii/framework/web/CController.php(1145): CFilter->filter(Object(CFilterChain))
#7 /var/www/yii/framework/web/filters/CInlineFilter.php(58): CController->filterAccessControl(Object(CFilterChain))
#8 /var/www/yii/framework/web/filters/CFilterChain.php(130): CInlineFilter->filter(Object(CFilterChain))
#9 /var/www/yii/framework/web/CController.php(291): CFilterChain->run()
#10 /var/www/yii/framework/web/CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#11 /var/www/yii/framework/web/CWebApplication.php(282): CController->run('rightanswer')
#12 /var/www/yii/framework/web/CWebApplication.php(141): CWebApplication->runController('questions/defau...')
#13 /var/www/yii/framework/base/CApplication.php(180): CWebApplication->processRequest()
#14 /var/www/yii/blog/index.php(13): CApplication->run()
#15 {main}
REQUEST_URI=/yii/blog/index.php/questions/rightanswer
HTTP_REFERER=http://localhost/yii/blog/index.php/questions/view/11
---
